odata-protocol/11 Data Service Requests.md

```diff
@@ -799,15 +804,14 @@ GET http://host/service/Products?$top=5
 ```
 :::
 
-If no unique ordering is imposed through an
-[`$orderby`](#SystemQueryOptionorderby) query option, the service MUST
-impose a stable ordering across requests that include `$top`.
-
 #### ##subsubsubsec System Query Option `$skip`
 
-The `$skip` system query option specifies a non-negative integer n that
-excludes the first n items of the queried collection from the result.
-The service returns items starting at position n+1.
+The `$skip` system query option specifies a non-negative integer $n$ that
+excludes the first $n$ items of the queried collection from the result.
+
+Let $A$ be a copy of the input set with a total order that extends any existing order of the input set but is otherwise chosen by the service. The total order MUST be stable across requests.
+
+The effect of ${\tt \$skip}=n$ is to exclude from the result the first $n$ instances in $A$. All remaining instances are kept in $A$ in the same order as they occur in $A$.
```

Comment:
Do we need to explain what happens if there are less than n instances in A? There is corresponding text in the $top section.